### DISCUSSION TITLE
Still booting into GNOME desktop after install

### DISCUSSION THREAD

**Discussion by donsequitur** [regolith-linux/regolith-desktop#872] [Q&A]
I'm on Ubuntu 22.04.  The install instructions end with 'System Restart', but I'm still being booted into GNOME desktop with no direction on what to do next (so I presume it must have been intended to automatically change my WM, but failed somehow)

**Comment by ajeshp** [comment]
When you log in, you need to select Regolith using the gear icon on the bottom right of the login screen. Otherwise, I believe, by default, it logs you into your previous DE.
